.templates/blynk_server/Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM adoptopenjdk/openjdk11
-MAINTAINER Graham Garner <garner.gc@gmail.com>
+MAINTAINER 877dev <877dev@gmail.com>
```

Comment:
Do you really want to become the maintainer? I should create an iotstack email, or have Andreas create one.

Reply:
@Slyke not not really :) I just put that there to differentiate my differences from the original. Feel free to change whatever you need to including port number quotations, I am just happy to see it getting updated and help a little!

Reply:
Can you please consider adding symlinks inside the /blynk folder and changing the ENTRYPOINT as shown above?
Result would be that the mail.properties file content will be displayed in the editor.

Reply:
Please change base image to 'adoptopenjdk/openjdk14'. According to #162